docs/source/analyze/cluster-resource.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Cluster Resource
+description: Analyze any attribute collected by the `ClusterResources` Collector
+---
+
+The `clusterResource` analyzer can be used to check any attribute of any known resources in the cluster in a generic manner.
+
+The `clusterResource` analyzer uses data from the [clusterResources collector](/collect/cluster-resources).
+The `clusterResources` collector is automatically added and is always present.
+
+You must specify the `kind` and `name` attributes. There is an optional `namespace` attribute to target the Kubernetes resource.
+
+The `yamlPath` attribute is used to specify a dot-delimited YAML path of a property on the Kubernetes resource referenced in `name`.
+The `when` attribute supports standard arithmetic comparison operators.
+
+The outcomes on this analyzer are processed in order, and execution stops after the first outcome that is truthy.
+
+## Parameters
+
+**checkName**: (Optional) Analyzer name.
+Used for uniqueness if multiple analyzers are defined with similar parameters.
+
+**kind**: (Required) The type of Kubernetes resource being targeted by `name`.
+
+**name**: (Required) The name of the resource to check.
+
+**namespace**: (Optional) The namespace to look in for the resource.
+
+**yamlPath**: (Required) The dot-delimited YAML path of a property on the Kubernetes resource.
+
+**regex**: (Optional) See [Regular Expression](/analyze/regex) documentation.
+**Note:** This is not supported when using arithmetic comparison in `when`.
+
+**regexGroups**: (Optional) See [Regular Expression](/analyze/regex) documentation.
+**Note:** This is not supported when using arithmetic comparison in `when`.
+
+## Outcomes
+
+The `when` value in an outcome of this analyzer can accept a few variations.
+
+If the `yamlPath` points to a quantity-based value (such as the size or quota or something), standard arithmetic comparison operators can be used: `<`, `<=`, `>`, `>=`, `==`, `!=`. 
```

Comment:
What is meant by "or something" here? Is it a value or can it be removed?

Reply:
can be removed, not a specific value

Reply:
sorted, I botched a merge commit but reverted it with a force push and fixed it again.

removed the `or something`